### PR TITLE
Project 3 Buisness site

### PR DESCRIPTION
**Pull request by Ann-Sofi**
In todays lecture we were informed to style the forms by class and not input... The forms on my site is styled by input but I hope it's okey this time. I will do it by class next time ;-) 

Regarding the hero-text, I did not use margin 0 and flexbox (like on the code session) to place it and some problems occurd on mobile version when the site was live. Is this type of code for hero-text a bad idea?

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -4,27 +4,127 @@
...
+        <div class="navigation-text">About us</div>
+      </a>
+      <a href="#"> 
+      <div class="navigation-text">Join us</div>
+      </a> 
+      <a href="#">    
+      <div class="navigation-text">Schedule</div>
+    </a>
+      </nav>
+    </header>
```
Comment (HIPPIEKICK) [inline code/index.html:26]: Remember to indent correctly, and possible omit the `div`s here for a cleaner structure. Try styling the links directly :)

File: code/index.html [hunk 2]
```diff
@@ -4,27 +4,127 @@
...
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+<!------ 2. HERO IMG AND TEXT-------->
+      <div class="hero"></div>
+        <picture>
+          <source srcset="./img/battleropes.jpg" media="(max-width: 767px)" />
+          <img class="hero-image" src="./img/highfive.jpg" alt="Hero image" />
+        </picture>
+        <h1 class="hero-text">"Strong people don't put others down, they lift them up"</h1>
+      </div>
```
Comment (HIPPIEKICK) [inline code/index.html:36]: You have two `</div>` here 👀

File: code/index.html [hunk 3]
```diff
@@ -4,27 +4,127 @@
...
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <h2 class="intro-text">Looking for a fun and varied type of training? Where you will learn new skills and build up both your strenght, condition as well as your self confidense.</h2> 
+    <p class="smaller-intro-text"">
+      Crossfit is training for life. You will lift, push, pull, jump, run - which all are things you will come accross in your ordinary life. In our box we train together, push each other and celebrate each others achievements. One of the best things about Crossfit is our community which is achived by awesome coaches and even more awesome members. Join us for the fun, the workouts and for the challanges! 
+    </p>
+
+
+    <!------ 4. SIGN UP FORM----->
+<div class="signup-container">
```
Comment (HIPPIEKICK) [inline code/index.html:48]: Indentation 😇

File: code/index.html [hunk 4]
```diff
@@ -4,27 +4,127 @@
...
+        <!--NAME,EMAIL-->
+        <label for="first-name">First Name &nbsp &nbsp </label>
+        <input type="text" id="first-name" name="first-name" style="width: 150px;" placeholder="Your first name" required>
+        <br><br>
+        <label for="last-name">Last Name &nbsp &nbsp</label>
+        <input type="text" id="last-name" name="last-name" style="width: 150px;" placeholder="Your last name" required>
+        <br><br>
+        <label for="email">Email adress</label>
+        <input type="email" id="email" name="email" style="width: 150px;" placeholder="Your email" required>
+        <br><br>
```
Comment (HIPPIEKICK) [inline code/index.html:60]: Nice structure! Try to omit the `br`s and use CSS styling to achieve the same thing with e.g. margin

File: code/index.html [hunk 5]
```diff
@@ -4,27 +4,127 @@
...
+        <input type="checkbox" name="group-training" id="group-training" value="group-traning">
+        <label for="group-traning">Group training</label>
+        <br>
+        <input type="checkbox" name="PT" id="PT" value="PT">
+        <label for="PT">PT</label>
+        <br>
+        <input type="checkbox" name="diet" id="diet" value="diet">
+        <label for="diet">Diet</label>
+        <br><br>
+      </div>
```
Comment (HIPPIEKICK) [inline code/index.html:80]: Nice with radio buttons and checkboxes 👍

File: code/index.html [hunk 6]
```diff
@@ -4,27 +4,127 @@
...
+
+
+          <!-- 6. FOOTER-->
     </section>
-    <!-- dont forget to link the javascript file here! -->
+      <footer class="footer">
+        <p>&copy Ann-Sofi Jönsson Technigo Bootcamp 2021</p>
+        <img class="footer-logo" src="./img/sport_barbell-512.png" alt="barbell">
+      </footer>
+    </section>
```
Comment (HIPPIEKICK) [inline code/index.html:124]: See if you can remove the wrapping `section` of the `footer`

File: code/script.js [hunk 7]
```diff
@@ -6,4 +6,8 @@ function toggle() {
 }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
+
 document.getElementById("section1").onclick = toggle
+document.getElementById("section2").onclick = toggle
+document.getElementById("section3").onclick = toggle
+document.getElementById("section4").onclick = toggle
```
Comment (HIPPIEKICK) [inline code/script.js:13]: Nice and clean 👍 You could also (if you want), break these out into variables, like this:
```
const section1 = document.getElementById("section-1")
section1.onclick = toggle
```

File: code/style.css [hunk 8]
```diff
@@ -1,7 +1,273 @@
...
+
+input[type=button], input[type=submit]:hover{
+background-color: white;
+}
+
+input[type=button], input[type=submit]:active {
+  background-color: white;
+  box-shadow: 0 4px #999;
+  transform: translateY(6px);
+}
```
Comment (HIPPIEKICK) [inline code/style.css:113]: I can't see you have any input elements of type button 👀

File: code/style.css [hunk 9]
```diff
@@ -1,7 +1,273 @@
...
+
+.hero-text {
+  font-size: 32px;
+  position: absolute;
+  top: 30%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: white; 
+  -webkit-text-stroke: 1px black;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:46]: Position absolute is another way of positioning text on top of another element, it seems to be working fine (referring to you mentioning you had issues on mobile).

**Review by HIPPIEKICK — APPROVED** [review]
Hi Ann-Sofi! Nice job on this project, I will approve it but when you have time check your indentation. If you need a recap -> [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e). Check out the [general feedback document](https://www.notion.so/technigo/Week-3-Business-site-9f3ef7b1c0ee480c8a10b9c7e460c475) as well for project specific guidelines :)

File: code/index.html (hunk 6 above)
Comment (Ann-Sofi) [reply code/index.html:124]: Yep, it worked and it looks the same :-)

File: code/style.css (hunk 8 above)
Comment (Ann-Sofi) [reply code/style.css:113]: Nope, I removed it :)

**Pull request closed without merging**